### PR TITLE
manage transaction and debtor fix

### PR DESCRIPTION
**Pull request by Douglasokolaa**
# Description

Paginated the tables, made the export button functional, replace all dummy data with actual value

### CODE REVIEW

File: app/Http/Controllers/DebtorController.php [hunk 1]
```diff
@@ -392,7 +238,7 @@ public function sendReminder(Request $request) {
 
         // $url = 'http://localhost:3000/debt' . '/send';
 
-        $url = env('API_URL', 'https://dev.api.customerpay.me') .'/debt'. '/send'; 
+        $url = env('API_URL', 'https://dev.api.customerpay.me') . '/debt' . '/send';
```
Comment (Jeremiahiro) [inline app/Http/Controllers/DebtorController.php:241]: this your concatenation is wonderful

File: app/Http/Controllers/DebtorController.php [hunk 2]
```diff
@@ -427,10 +271,17 @@ public function sendReminder(Request $request) {
             Log::info('Catch error: LoginController - ' . $e->getMessage());
```
Comment (Jeremiahiro) [inline app/Http/Controllers/DebtorController.php:271]: LoginController?

File: app/Http/Controllers/DebtorController.php [hunk 3]
```diff
@@ -442,7 +293,8 @@ public function sendReminder(Request $request) {
         }
     }
 
-    public function sheduleReminder(Request $request) {
+    public function sheduleReminder(Request $request)
```
Comment (Jeremiahiro) [inline app/Http/Controllers/DebtorController.php:296]: is this supposed to be `scheduleReminder?`

File: app/Http/Controllers/DebtorController.php [hunk 4]
```diff
@@ -452,66 +304,69 @@ public function sheduleReminder(Request $request) {
...
-                        'transaction_id' => $request->transaction_id,
-                        'message' => $request->message,
-                    ],
-                ];
-                $response = $client->request("POST", $url, $payload);
-
-                $statusCode = $response->getStatusCode();
-                $body = $response->getBody();
-                $data = json_decode($body);
+        $url = env('API_URL', 'https://dev.api.customerpay.me') . '/debt' . '/schedule';
```
Comment (Jeremiahiro) [inline app/Http/Controllers/DebtorController.php:307]: this can work without the concatenation

File: app/Http/Controllers/DebtorController.php [hunk 5]
```diff
@@ -452,66 +304,69 @@ public function sheduleReminder(Request $request) {
...
+                Session::flash('message', $data->Message);
+
+                return back();
+            } else {
+                $request->session()->flash('alert-class', 'alert-success');
+                // Session::flash('message', $data->message);
+                return redirect()->back();
+            }
+        } catch (RequestException $e) {
+            Log::info('Catch error: LoginController - ' . $e->getMessage());
```
Comment (Jeremiahiro) [inline app/Http/Controllers/DebtorController.php:338]: "LoginController"

File: app/Http/Controllers/DebtorController.php (hunk 1 above)
Comment (Douglasokolaa) [reply app/Http/Controllers/DebtorController.php:241]: looks correct any hint?
@Jeremiahiro

File: app/Http/Controllers/DebtorController.php (hunk 2 above)
Comment (Douglasokolaa) [reply app/Http/Controllers/DebtorController.php:271]: Some codes i met, i will adjust ASAP

**Pull request merged**